src/ngraph/event_tracing.cpp

```diff
@@ -19,72 +19,191 @@
 #include <string>
 
 #include "event_tracing.hpp"
+#include "ngraph/log.hpp"
 #include "nlohmann/json.hpp"
 
 using namespace std;
+using namespace ngraph;
 
 static bool read_tracing_env_var()
 {
-    return (std::getenv("NGRAPH_ENABLE_TRACING") != nullptr);
+    static const bool is_enabled = (getenv("NGRAPH_ENABLE_TRACING") != nullptr);
```

Comment:
Longer term it would make more sense to have a way for a bridge to turn this on and off.

Reply:
👍

Reply:
There are already calls to do this
```
ngraph::event::Manager::enable_event_tracing()
ngraph::event::Manager::disable_event_tracing()
```